tests/test_google_sheets_datatypes.py

```diff
@@ -155,7 +155,7 @@ def test_run(self):
         # Verify that all data has been coerced to the expected column datatype
         record_data = [message['data'] for message in synced_records[test_sheet]['messages'] if message.get('data')]
         data_map = {
-	    "Currency": Decimal, # BUG Currency is being identified as a decimal type rather than string https://jira.talendforge.org/browse/TDL-14360
+	    "Currency": str, # BUG Currency is being identified as a decimal type rather than string https://jira.talendforge.org/browse/TDL-14360
```

Comment:
will commented bug be resolved in this PR?

Reply:
Yes, removed the comment for the same